waspc/packages/deploy/src/providers/fly/helpers/helpers.ts

```diff
@@ -38,8 +42,11 @@ function getWaspBuildDir(waspProjectDir: string) {
 	return path.join(waspProjectDir, '.wasp', 'build');
 }
 
-export function ensureDirsInCmdAreAbsoluteAndPresent(thisCommand: Command): void {
-	const waspProjectDirPath: string | undefined = thisCommand.opts().waspProjectDir;
+export function ensureDirsInCmdAreAbsoluteAndPresent(
+	thisCommand: Command,
```

Comment:
Oh wow we are using tabs here for indentation? I see now they were also used before? Uff.